test/e2e/auth/service_accounts.go

```diff
@@ -539,7 +581,7 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 		}
 	})
 
-	ginkgo.It("should support InClusterConfig with token rotation [Slow] [Feature:TokenRequestProjection]", func() {
+	ginkgo.It("should support InClusterConfig with token rotation [Slow]", func() {
```

Comment:
do you know what test suite this runs in (if any)?

Reply:
`pull-kubernetes-e2e-kind` will run this when removed the `[Feature:TokenRequestProjection]`.

Reply:
`Slow` tests get skipped in presubmit

https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml#L32

Reply:
right. i meant `ginkgo.It("should set ownership and permission when RunAsUser or FsGroup is present [LinuxOnly] [NodeFeature:FSGroup] [Feature:TokenRequestProjection]"` and the newly added one.

i'll drop `Slow` because it is not that slow when i was testing it.

Reply:
if remove the `Slow` then we might not need the newly added e2e test

Reply:
> i'll drop `Slow` because it is not that slow when i was testing it.

it takes at least 10 minutes to observe a rotation, right? that's... slow

Reply:
i think it will show up in https://k8s-testgrid.appspot.com/sig-auth-gce#gce-slow after remove this